orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -391,19 +423,12 @@ def _get_forward_graph(module, *inputs, **kwargs):
         f = io.BytesIO()
 
         # Deepcopy inputs, since input values may change after model run.
-        sample_inputs_copy = copy.deepcopy(inputs)
-
-        # Ignore optional *inputs explicitly specified as None
-        sig = signature(module.forward)
-        all_input_names = sig.parameters.keys()
-        input_names = []
-        dynamic_axes = {}
-        for input_idx, name in enumerate(all_input_names):
-            if input_idx < len(inputs) and inputs[input_idx] is not None:
-                input_names.append(name)
-                dynamic_axes[name] = {}
-                for dim_idx in range(len(inputs[input_idx].shape)):
-                    dynamic_axes[name].update({dim_idx : f'input{input_idx}_dim{dim_idx}'})
+        # Inputs may contain tensors that have attributes preventing their deepcopy (example grad_fn).
+        # Deepcopy only the data component of the input tensors for export.
```

Comment:
How about making this comment a little bit more "scary" by prepending IMPORTANT: or WARNING: or NOTE: or something like that? This change may have side effects, so better give a clue for future us